### PR TITLE
Added two floating actions buttons to get data from remote or local database

### PR DESCRIPTION
**Pull request by Faka-GLB**
Implemented a floating action button to get the characters from API service, and another to get from the Realm database.
Try again button from the error state removed, replaced with the two FABs.

Nexus 5X:
image

Nexus S:
image

Error state:
image

Service load:
https://user-images.githubusercontent.com/84337198/132877884-49fbe20f-0474-4915-b696-d8dc6dc8a0e6.mp4

Database load:
https://user-images.githubusercontent.com/84337198/132877576-93b36771-8470-4e44-aaf7-463d77087d48.mp4

App module:
data module marvel app

The other modules didn't had any file added or removed

### CODE REVIEW

File: app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt [hunk 1]
```diff
@@ -51,39 +50,49 @@ class MainActivity : AppCompatActivity(), CharacterClickListener {
...
         Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
     }
 
-    private fun setListener() {
-        binding.buttonMainActivityTryAgain.setOnClickListener {
-            viewModel.getCharacters()
+    private fun setListeners() {
+        binding.floatingActionButtonMainActivityRemote.setOnClickListener {
+            viewModel.getCharacters(fromRemote = true)
+            loadingState()
```
Comment (BQuinteros) [inline app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt:92]: you do not have to call loadingstate. The viewmodel calls states

Comment (Faka-GLB) [reply app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt]: removed

File: app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt [hunk 2]
```diff
@@ -51,40 +50,48 @@ class MainActivity : AppCompatActivity(), CharacterClickListener {
...
+        binding.progressBarMainActivity.visibility = View.GONE
         binding.recyclerViewMainActivity.visibility = View.VISIBLE
     }
 
     private fun errorState() {
-        binding.groupBaseView.visibility = View.GONE
-        binding.buttonMainActivityTryAgain.visibility = View.VISIBLE
+        binding.groupBaseView.visibility = View.VISIBLE
+        binding.progressBarMainActivity.visibility = View.GONE
+        binding.floatingActionButtonMainActivityLocal.show()
```
Comment (mauroLonghi) [inline app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt:71]: pls, if you use this method, be consistent with the rest of the components that show or hide its visibility

Comment (Faka-GLB) [reply app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt:71]: The show() and hide() methods of the FABs are used to animate their visibility

Comment (mauroLonghi) [reply app/src/main/java/com/onboarding/marvel_mvvm/activity/MainActivity.kt:71]: perfect

**Review by mauroLonghi — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**